src/toil/jobStores/googleJobStore.py

```diff
@@ -113,12 +113,12 @@ def destroy(self):
                     pass
 
     def create(self, jobNode):
-        jobStoreID = self._newID()
-        job = JobGraph(jobStoreID=jobStoreID, unitName=jobNode.name, jobName=jobNode.job,
-                       command=jobNode.command, remainingRetryCount=self._defaultTryCount(),
-                       logJobStoreFileID=None, predecessorNumber=jobNode.predecessorNumber,
-                       **jobNode._requirements)
+        job = self.getJobGraph(jobNode)
         self._writeString(jobStoreID, cPickle.dumps(job, protocol=cPickle.HIGHEST_PROTOCOL))
+        if hasattr(self, "_batchedJobGraphs") and self._batchedJobGraphs is not None:
+            self._batchedJobGraphs.append(job)
+        else:
+            self._writeString(jobStoreID, cPickle.dumps(job, protocol=cPickle.HIGHEST_PROTOCOL))
```

Comment:
Where is the context manager? Does Google do batch creation? This code is broken at the moment. Maybe just make a note to add the batch ability if Google has it.